client/callopt/options.go

```diff
@@ -193,6 +196,24 @@ func WithRetryPolicy(p retry.Policy) Option {
 	}}
 }
 
+// WithFallback is used to set the fallback policy for a RPC call.
+// Below is use demo, eg:
+//
+//	demo1. call with fallback for error
+//		`resp, err := cli.Mock(ctx, req, callopt.WithFallback(fallback.ErrorFallback(yourFBFunc))`
+//
+//	demo2. call with fallback for error and enable reportAsFallbackRet, reportAsFallbackRet set true the RPC stat will set error as fallback not original error
```

Comment:
the comment here is not easy to understand, and I'd suggest:

> , which sets reportAsFallbackRet to be true and replaces the original error in RPC stat to be FallbackError (code=XXX)

Reply:
done